action.py

```diff
@@ -26,6 +27,7 @@ def __init__(
     ):
         self._base_url = base_url
         self._channel_id = channel_id
+        self.lang = lang
```

Comment:
For consistency, this can be named `self._lang` rather than `self.lang` (marking it as intended to be a private field)